README.md

```
@@ -93,21 +175,28 @@ Command-line argument parser with Builder pattern.
 $ cd libvalacore
-$ meson build
-$ cd build
-$ ninja
-$ sudo ninja install
+$ meson setup build
+$ ninja -C build
+$ sudo ninja -C build install
 ```
 
 # How to test
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)